### PR TITLE
Simplify installation - Do not install Istio nor NGNIX-ingress

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: content/docs/develop/continuous_delivery/expose_services/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    ```console
+    kubectl apply configmap -n keptn ingress-config --from-literal=ingress_hostname_suffix=${INGRESS_IP} --from-literal=ingress_port=${INGRESS_PORT} --from-literal=ingress_protocol=${INGRESS_PROTOCOL} --from-literal=ingress_gateway=${INGRESS_GATEWAY}
+    ```
+
+* If you have already set up a domain that points to your Istio ingress, you can use it for the `INGRESS_HOSTNAME_SUFFIX`. In this case, use the following command to create the `ingress-config` ConfigMap in the `keptn` namespace:
+
+    ```
+    INGRESS_HOSTNAME=<YOUR_HOSTNAME>
+    INGRESS_PORT=<PORT_OF_YOUR_INGRESS> 
+    INGRESS_PROTOCOL=<PROTOCOL>                            # "http" or "https"
```
Comment (agrimmer) [inline content/docs/develop/continuous_delivery/expose_services/index.md:74]: Here the `INGRESS_GATWAY` is missing.

File: content/docs/develop/continuous_delivery/expose_services/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    kubectl apply configmap -n keptn ingress-config --from-literal=ingress_hostname_suffix=${INGRESS_HOSTNAME} --from-literal=ingress_port=${INGRESS_PORT} --from-literal=ingress_protocol=${INGRESS_PROTOCOL}
+    ```
+
+* After creating the ConfigMap, restart the `helm-service`:
+
+    ```
+    kubectl delete pod -n keptn -lrun=helm-service
+    ```
+
+* If you are on OpenShift, also restart the `openshift-route-service`:
```
Comment (agrimmer) [inline content/docs/develop/continuous_delivery/expose_services/index.md:87]: It is not required to delete the `openshift-route-service` pod.

File: content/docs/develop/continuous_delivery/expose_services/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/develop/continuous_delivery/expose_services/index.md:74]: Added `INGRESS_GATEWAY`.

File: content/docs/develop/continuous_delivery/expose_services/index.md (hunk 2 above)
Comment (johannes-b) [reply content/docs/develop/continuous_delivery/expose_services/index.md:87]: Okay, removed it.

File: content/docs/develop/continuous_delivery/expose_services/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    ```
+    
+    ```console
+    kubectl apply configmap -n keptn ingress-config --from-literal=ingress_hostname_suffix=${INGRESS_HOSTNAME} --from-literal=ingress_port=${INGRESS_PORT} --from-literal=ingress_protocol=${INGRESS_PROTOCOL}
+    ```
+
+* After creating the ConfigMap, restart the `helm-service`:
+
+    ```
+    kubectl delete pod -n keptn -lrun=helm-service
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/develop/continuous_delivery/expose_services/index.md:84]: thx for the solution.

File: content/docs/develop/continuous_delivery/expose_services/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+* [Determine the ingress IP and ports](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports):
+
+    ```console
+    kubectl -n istio-system get svc istio-ingressgateway
+    ```
+
+* Create the `ingress-config` ConfigMap in the `keptn` namespace:
+
+    ```
+    INGRESS_IP=<IP_OF_YOUR_INGRESS>.xip.io
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/develop/continuous_delivery/expose_services/index.md:57]: Renamed it.

**Pull request merged**